src/FSharpPlus/Cont.fs

```diff
@@ -3,6 +3,7 @@
 /// <summary> Computation type: Computations which can be interrupted and resumed.
 /// <para/>   Binding strategy: Binding a function to a monadic value creates a new continuation which uses the function as the continuation of the monadic computation.
 /// <para/>   Useful for: Complex control structures, error handling, and creating co-routines.</summary>
+[<Struct>]
```

Comment:
Cont have `NoEquality; NoComparison` ?

Reply:
Since it wraps a function type?

Reply:
Yes, but I think it doesn't require it explicitly. We can certainly added it in anycase.

Reply:
Usually f# compiler emits warnings when I do similar type declarations... you know the compiler better :wink:

Reply:
Not really. I know better some specific parts only ;) 
But yes, it started to emit some warnings about it, for some reason I'm still not sure (have to read again the warning message) it doesn't emit it for ``Cont`` and some other wrappers. I should add to the roadmap something like checking what other attributes are still missing (ie: CompiledName, Representation, and so on)